helm/resource_release.go

```diff
@@ -889,11 +908,24 @@ func logValues(values map[string]interface{}, d resourceGetter) error {
 	return nil
 }
 
-func getRelease(cfg *action.Configuration, name string) (*release.Release, error) {
+func getRelease(m *Meta, cfg *action.Configuration, name string) (*release.Release, error) {
+	debug("%s getRelease wait for lock", name)
+	m.Lock()
+	defer m.Unlock()
+	debug("%s getRelease got lock, started", name)
+
+	cfg.Log = debug
```

Comment:
Can you help me understand why we had to set this and the one below?

This should get set to `debug` in [GetHelmConfiguration](https://github.com/hashicorp/terraform-provider-helm/blob/master/helm/provider.go#L319) which in turn should set it on Releases.Log in the helm actions package [here](https://github.com/helm/helm/blob/9b42702a4bced339ff424a78ad68dd6be6e1a80a/pkg/action/action.go#L232).

Reply:
Oops, this is a leftover from my debugging session - thanks for pointing that out!